tensorflow/tensorboard
# cleanup: Remove test deps on //@npm//@angular/compiler.

Remove seemingly unnecessary deps from test libraries on `//@npm//@angular/compiler`. 

Why these subset of tests have this dependency is unknown to me - none of the corresponding test files actually `import @angular/compiler;`.  Looking at the generated bundles for these tests, the output appears only partially compiled (ie the bundled output is more akin to `//tensorboard:dev` than `//tensorboard`), but if the angular compiler is actually included by the test runtime then it is done somewhere deep in the test framework. Not sure if it is Karma or spec_bundle() or elsewhere.

To test:
* Run the tests. They pass.
* Check I got all the instances in tests:
  ```
  $ grep -r "angular/compiler" *
  defs/BUILD:# Custom ts_library compiler that runs tsc_wrapped with angular/compiler-cli statically linked
  defs/BUILD:        "@npm//@angular/compiler-cli",
  webapp/dev_assets/BUILD:        "@npm//@angular/compiler",
  webapp/dev_assets/main_dev.ts:import '@angular/compiler';
  ```
